exercises/08.rate-limiting/01.solution.basic/server/index.ts

```diff
@@ -74,12 +74,12 @@ app.use(morgan('tiny'))
 // When running tests or running in development, we want to effectively disable
 // rate limiting because playwright tests are very fast and we don't want to
 // have to wait for the rate limit to reset between tests.
-const maxMultiple = process.env.TESTING ? 10_000 : 1
+const limitMultiple = process.env.TESTING ? 10_000 : 1
```

Comment:
I changed max->limit for the rate-limiting express library, because it said that 'max' field has been deprecated and replaced by 'limit'. It worked for me using 'limit' instead of 'max'

Reply:
Thanks for this!